package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "perry-white",
-  "version": "0.3.2",
+  "version": "0.3.3",
   "subversion": "1",
```

Comment:
FYI, you don't need to bump version.  I use https://github.com/sindresorhus/np#readme to publish which will auto-bump the version

Reply:
👍